src/content/docs/victory-histogram.md

```diff
@@ -0,0 +1,559 @@
+```jsx
+domainPadding={{x: [10, -10], y: 5}}
+```
+
+## eventKey
+
+`type: string || integer || array[string] || function`
+
+`VictoryHistogram` uses the standard `eventKey` prop to specify how event targets are addressed. **This prop is not commonly used.** [Read about the `eventKey` prop in more detail here](/docs/common-props#eventkey)
+
+```jsx
+eventKey = "x";
+```
+
+## events
+
+`type: array[object]`
+
+`VictoryHistogram` uses the standard `events` prop. [Read about it here](/docs/common-props#events)
+
+See the [Events Guide][] for more information on defining events.
+
+```playground
+<div>
+  <h3>Click Me</h3>
+  <VictoryHistogram
+    style={{
+      data: { fill: "#c43a31" }
+    }}
+    events={[{
+      target: "data",
+      eventHandlers: {
+        onClick: () => {
+          return [
+            {
+              target: "data",
+              mutation: (props) => {
+                const fill = props.style && props.style.fill;
+                return fill === "black" ? null : { style: { fill: "black" } };
+              }
+            }
+          ];
+        }
+      }
+    }]}
+    data={sampleHistogramData}
+  />
+</div>
+```
+
+## externalEventMutations
+
+`type: array[object]`
+
+`VictoryHistogram` uses the standard `externalEventMutations` prop. [Read about it in detail](/docs/common-props#externalEventsMutations)
+
+## groupComponent
+
+`type: element`
+
+`VictoryHistogram` uses the standard `groupComponent` prop. [Read about it here](/docs/common-props#groupcomponent)
+
+_default:_ `<g/>`
+
+```jsx
+groupComponent={<g transform="translate(10, 10)" />}
+```
+
+## height
+
+`type: number`
+
+`VictoryHistogram` uses the standard `height` prop. [Read about it here](/docs/common-props#height)
+
+_default (provided by default theme):_ `height={300}`
+
+```jsx
+height={400}
+```
+
+## horizontal
+
+`type: boolean`
+
+The horizontal prop determines whether the bins will be laid vertically or horizontally. The bins will be vertical if this prop is false or unspecified, or horizontal if the prop is set to true.
+
+_default:_ horizontal={false}
+
+```playground
+<VictoryChart
+  theme={VictoryTheme.material}
+  domainPadding={{ x: 10 }}
+>
+  <VictoryHistogram horizontal
+    style={{
+      data: { fill: "#c43a31" }
+    }}
+    data={sampleHistogramData}
+  />
+</VictoryChart>
+```
+
+## labelComponent
+
+`type: element`
+
+`VictoryHistogram` uses the standard `labelComponent` prop. [Read about it here](/docs/common-props#labelcomponent)
+
+_default:_ `<VictoryLabel/>`
+
+```playground
+<VictoryHistogram
+  data={sampleHistogramData}
+  labels={({ datum }) => datum.y}
+  style={{ labels: { fill: "white" } }}
+  labelComponent={<VictoryLabel dy={30}/>}
+/>
+```
+
+## labels
+
+`type: array || function`
+
+`VictoryHistogram` uses the standard `labels` prop. [Read about it here](/docs/common-props#labels)
+
+```playground
+<VictoryHistogram
+  data={sampleHistogramData}
+  labels={({ datum }) => `Total binned values: ${datum.y}`}
```

Comment:
The labels in this example are pretty long and they end up looking weird. These would look nicer with shorter labels or with a newline